src/Previewer/Previewer.js

```diff
@@ -90,11 +90,13 @@ export default function Previewer({
     let text = panel_ui.xml.trim();
     let target_id;
     let tree;
+    // let template;
 
     try {
       tree = ltx.parse(text);
-      [target_id, text] = targetBuildable(tree);
+      [target_id, text /* template */] = targetBuildable(tree);
```

Comment:
we need to expose `workbench.template` for `Internal.js`

for `External.js` we probably need to write this to disk